src/main/java/bob/command/FindCommand.java

```diff
@@ -0,0 +1,37 @@
+package bob.command;
+
+import bob.exception.BobException;
+import bob.exception.BobInvalidTaskNumberException;
+import bob.storage.StorageFile;
+import bob.task.TaskList;
+import bob.ui.TextGenerator;
+import bob.ui.TextUi;
+
+/**
+ * The FindCommand Class encapsulates logic that can be executed
+ * to find tasks with descriptions corresponding to a given keyword.
+ */
+public class FindCommand extends Command {
+
+    private String keyword;
```

Comment:
Some of your classes have a spacing in between the class and fields whereas other doesn't. Keep it standardised.